streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -92,7 +92,11 @@
     private final Map<TaskId, StandbyTask> standbyTasks;
     private final Map<TopicPartition, StreamTask> activeTasksByPartition;
     private final Map<TopicPartition, StandbyTask> standbyTasksByPartition;
+    private final Map<TopicPartition, StreamTask> suspendedTasksByPartition;
```

Comment:
`suspendedStandbyTasksByPartition` and `suspendedTasksByPartition` maps are not used anywhere?